workspaces/jupyterlab-python/requirements.txt

```diff
@@ -3,6 +3,7 @@ beautifulsoup4==4.12.2
 bokeh==3.2.2
 colormath==3.0.0
 defusedxml==0.7.1
+Faker==19.6.2
```

Comment:
@nwalters512 slight piggyback but followup to here: https://github.com/PrairieLearn/PrairieLearn/pull/8693